docs/02-kepler-installation-guide.md

```diff
@@ -0,0 +1,100 @@
+# Kepler Installation Guide
+
+This guide will show you how to install Kepler, a lightweight pod-level power consumption metrics exporter, on a Minikube cluster running on an Ubuntu 22.04 host.
+
+## Prerequisites
+
+Before you begin, you'll need to have the following:
+
+- Kernel 4.18+
+- Docker installed
+- Make and go packages installed
+- kubectl command-line tool installed
+- Helm command-line tool installed
+
+## Installation steps
+
+Start by installing Minikube on your machine. You can follow the instructions in the [Minikube documentation](https://minikube.sigs.k8s.io/docs/start/).
+I ran the following commands:
+
+```bash
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+sudo install minikube-linux-amd64 /usr/local/bin/minikube
+sudo usermod -aG docker $USER && newgrp docker
+```
+
+Once you have Minikube installed, start a new cluster using the following command (CPUs and Memory values can be customized):
+
+```bash
+# check CPU cores
+lscpu | grep "CPU(s)"
+# start the cluster
+minikube start --cpus 4 --memory 4096
+```
+
+Next, install the Prometheus Operator on your cluster by running the following commands:
+
+```bash
+git clone https://github.com/prometheus-operator/kube-prometheus.git
+cd kube-prometheus/
+kubectl apply --server-side -f manifests/setup
+kubectl wait --for condition=Established --all CustomResourceDefinition	--namespace=monitoring
+kubectl apply -f manifests/
+cd ../
+```
+
+Clone Kepler repository and generate manifest files by running the following commands:
+
+```bash
+git clone https://github.com/sustainable-computing-io/kepler.git -b v0.4
+cd kepler/
+# to configure Prometheus to scrape Kepler-exporter endpoints, Kepler exporter servicemonitor object is required
+make build-manifest OPTS="PROMETHEUS_DEPLOY"

```

Comment:
How did you guys applied make on mac? I tried to install it via brew but it is not working for me

Reply:
strange, `brew install make` should do it.. 🤔 did you get any errors?